src/telemetry/telemetry.h

```diff
@@ -9,7 +9,4 @@
 #pragma once
 
 /* initialise the telemetry component */
-int init_telemetry( void );
-
-/* queue up a message to be sent to ground */
-void send_telemetry( void );
```

Comment:
The design idea here is that only the `telemetry` module is called from outside and not its children. `telemetry` would therefore need to delegate the "work" down to its children through functions like this. This is to avoid unnecessary conflicts between local functions such as `read_queue` in `downlink_queue` and in `img_processing/data_queue` since these would only be included in their respective parents and therefore never together.

Reply:
Ok, now I see the reason for the design. I agree to all that you wrote, I'll include it in the next commit.

Reply:
I didn't like the idea of having something like:
```c
int send_telemetry_local(int d, int p) {
    push(&downlink_queue, d, p);
    return SUCCESS;
}
```
in `downlink_queue.c` and then:
```c
int send_telemetry( d, p ) {
    return send_telemetry_local(d, p);
}
```
in `telemetry.c`, as it seems redundant, but I can agree that it helps with avoiding naming and import conflicts.

Reply:
Perhaps you can either rename `push` to `send_telemetry_local` or replace the latter with the first to avoid one step. Either way, the function in `downlink_queue.c` won't be accessible from outside of `telemetry` and its children.